### PR TITLE
Enhance site SEO and accessibility

### PR DESCRIPTION
**Pull request by Olaleye-Blessing**
- improved accessibility of navbar hamburger
- improved social links accessibility
- added robots.txt file to restrict crawlers from indexing certain pages

### Fixes Issue
Closes: #324

### CODE REVIEW

**Review by JakubChorzepa — COMMENTED** [review]
We could also add custom 404 page with redirect to home page button. It would improve SEO.

**Comment by Olaleye-Blessing** [comment]
I don't think there is a design for a 404 page yet.

**Comment by rohitdasu** [comment]
> I don't think there is a design for a 404 page yet.

hey, i agree we don't have any design but you can create a sample 404 page. it's better than using the default 404 page which vercel/nextjs shows

File: components/Topbar/Topbar.tsx [hunk 1]
```diff
@@ -22,22 +22,24 @@ export const Topbar = () => {
...
   };
   const SUBMIT_PAGE = '/projects/submit';
+  const NON_DESKTOP_ROUTES = ['/', '/404'];
   return (
     <div className="z-50 border-b border-gray-800 bg-gray-900">
       <nav className="m-auto flex h-[5.46rem] max-w-screen-xl justify-between p-4 py-5">
         <div className="flex items-center gap-10">
           <Logo />
-          {router.pathname !== '/' && <DesktopRoutes />}
+          {!NON_DESKTOP_ROUTES.includes(router.pathname) && <DesktopRoutes />}
```
Comment (rohitdasu) [inline components/Topbar/Topbar.tsx:31]: could you please rename it MOBILE_ROUTES? non_desktop_routes kinda makes it too long imo

**Review by JakubChorzepa — APPROVED** [review]
LGTM

**Review by rohitdasu — APPROVED** [review]

**Pull request merged**